mantidproject/mantid
# Link in frequency tab correctly

**Description of work.**

This PR updates the transform tab to use the main context rather than it's own separate context. The changes here are minimal as the moving of the phase table calculations described in #25511 is being done in a separate PR.



**To test:**

* Check that both the old and new Frequency Domain Analysis interfaces open.
* In the new FDA interface load MUSR 22725 (This should be available from the unit test directory)
* Go to the transform tab and click calculate FFT. Plot the resultant workspace from the ADS. It should have three spectrum all of which have a peak around 1.7.
* In the transform tab switch the combo box at the top to Maxent click calculate max-ent. Plot the resultant workspace and check it looks sensible. 



Fixes #25306  



*This does not require release notes* because this is an internal refactoring.

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
